src/migrations/20221810114644-add-suggest-changes-table.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+CREATE TABLE IF NOT EXISTS suggest_change_set (
+  id serial primary key,
+  environment varchar(255) NOT NULL,
+  state varchar(255) NOT NULL,
+  project varchar(255) NOT NULL,
```

Comment:
Should project and environment be also foreign keys? What will happen to suggest changes when project is deleted?

Reply:
Adding it makes sense however, we are already at 5 joins for the query, I am reluctant to add more

Reply:
We can always performance test this and make changes later? Having to manually clean up if an environment / project gets deleted sounds like a nightmare to me.

Reply:
Yea, I don't think the performance hit will be something remarkable. Those joins on foreign keys are quite fast.

Reply:
Provided you add index on elements that you do FK joins on

Reply:
Primary keys are indexed by default.